daemon/api_snaps.go

```diff
@@ -463,12 +463,10 @@ func snapInstall(ctx context.Context, inst *snapInstruction, st *state.State) (*
 		return nil, errors.New(i18n.G("cannot install snap with empty name"))
 	}
 
-	var ckey string
 	if inst.CohortKey == "" {
 		logger.Noticef("Installing snap %q revision %s", inst.Snaps[0], inst.Revision)
 	} else {
-		ckey = strutil.ElliptLeft(inst.CohortKey, 10)
-		logger.Noticef("Installing snap %q from cohort %q", inst.Snaps[0], ckey)
+		logger.Noticef("Installing snap %q from cohort %q", inst.Snaps[0], strutil.ElliptLeft(inst.CohortKey, 10))
```

Comment:
why this change? this goes into a log

Reply:
There wasn't any change to the log here (unless I'm misreading something), this was just a result of the change below (no more `ckey` variable).